### ISSUE TITLE
Rule E721 false positive on member functions

### ISSUE THREAD

**Issue by da-raf** [LintlyCI/Flake8Rules#62]
```python
class A:
    def type(self, b):
        return "fish"


a = A()
if a.type("test") == "fish":
    print("yes")
else:
    print("no")
```
Equality operator comparison of the result of a member function called "type" with a single parameter in addition to `self` is reported as a violation of E721, which forbids comparison of type objects, although the return type of the function is not a type-object.
A real-world example of this is the "type" function of the Canvas class in tkinter, which returns strings.

**Comment by da-raf** [comment]
sorry, bug belongs to pycodestyle
